data/geany.glade

```diff
@@ -7726,7 +7726,7 @@
                                   <object class="GtkRadioMenuItem" id="cr">
                                     <property name="visible">True</property>
                                     <property name="can_focus">False</property>
-                                    <property name="label" translatable="yes">Convert and Set to CR (_Mac)</property>
+                                    <property name="label" translatable="yes">Convert and Set to CR (_Mac OS 9)</property>
```

Comment:
Didn't you mean to label these "Classic Mac" or something? https://github.com/geany/geany/pull/396#issuecomment-73346068 https://github.com/geany/geany/pull/396#issuecomment-73358735